### ISSUE TITLE
Emphasis the limited scope of the questioning and discussion for the chat bot

### ISSUE THREAD

**Issue by SimonMcCallum** [klAndersen/IMT4601_RPP#1]
General Turing test is not within scope. You need to make sure every time you mention the Turing test you should make sure you say "limited Turing test"

**Comment by SimonMcCallum** [comment]
More references.  You should probably add some references to ACM SIGCSE. There are a massive collection of papers on learning programming.

**Comment by SimonMcCallum** [comment]
I have the formulation where you refer to a paper by the number.  You should be able to read a sentence ignoring everything in brackets.  e.g. I have this "the paper by [3] were they used" use the name,  The new thesis standard will use citet for this but do not worry to much in this report as the markers are probably not as concerned about this as I am.

**Comment by klAndersen** [comment]
In regards to the numbered references, this is the template setup from Einar. He uses an outdated bibtex called unsrt.

**Comment by SimonMcCallum** [comment]
Yep that is all changing.  But it is not the numbering but more that in the sentence above you should say something like "the paper by Crutzen et. al. [3] were they used"

**Comment by SimonMcCallum** [comment]
The numbers are completely silent and so you have to name the authors.

**Comment by SimonMcCallum** [comment]
1.5 point 6 should be a limited Turing test

**Comment by SimonMcCallum** [comment]
using AI and do research on the -> using AI and *to* do research on the

**Comment by SimonMcCallum** [comment]
mentioned in **many studies** [3,4,5.........]

**Comment by SimonMcCallum** [comment]
Section 2.2 the second para is too long and contains two different topics. break this up, and bring the main topic to the front of the paragraph

**Comment by SimonMcCallum** [comment]
Finish 2.3 there should be some quick stuff here
 2.4 Passing a limited Turing test.  A test within a specific domain area. look at success in limited domains of knowledge.
and 2.5 I would argue that there are many different types of question. There are structured queries, where can be object, which query to longer to process.  The next layer is information gathering questions.  Questions designed to get information.

**Comment by SimonMcCallum** [comment]
4.3.1 the subversions should be incrementing a little
. May could be V0.2

**Comment by SimonMcCallum** [comment]
Generally looking reasonable. Might need some more details on some of the projects but you are limited for time and resources

**Comment by klAndersen** [comment]
Yeah, the reason for that was that I was under the impression that we had to write 4 versions of the the hour plan to get approved; quote p. 11 in the template:
"For each deliverable, identify 4 versions, having an ’increasing’ degree of completeness/
quality"

**Comment by klAndersen** [comment]
I was also unsure whether or not it was relevant to add the details to which how the answer would be displayed by use of AI. My plan is instead of showing every answer, that all answers are compiled into one, so that the answer will be a "sum of all on the given page". 

E.g. if there are 4 answers (and all have vote >= 0), then unless they contain the same text, it could be compiled into a full answer text which is shown to the user. However, I did not find a "fitting" location to put it, so I therefore choose to omit it from the report. Not sure if this type of detail should be included?

**Comment by SimonMcCallum** [comment]
More detail on how you plan to analyse the results. What kind of questions
you will ask.  How you would interpret particular answers.
